pimd/pim_cmd.c

```diff
@@ -7274,13 +7274,13 @@ DEFUN (ip_pim_rp_keep_alive,
 
 DEFUN (no_ip_pim_rp_keep_alive,
        no_ip_pim_rp_keep_alive_cmd,
-       "no ip pim rp keep-alive-timer (31-60000)",
+       "no ip pim rp keep-alive-timer [(1-65535)]",
        NO_STR
        IP_STR
        "pim multicast routing\n"
        "Rendevous Point\n"
        "Keep alive Timer\n"
-       "Seconds\n")
+       IGNORED_IN_NO_STR)
 {
 	const char *vrfname;
 	char rp_ka_timer[5];
```

Comment:
Same here: either stringnify define or use `DESTROY` in the YANG op.

Reply:
changed to setting to 3*candidate config suppress time + probe time -- with larger str buffer